### ISSUE TITLE
D1 Co-Op Fusion Does Multiplayer Damage Levels

### ISSUE THREAD

**Issue by CDarrow** [CDarrow/DXX-Retro#24]
In D1 single player, fusion does twice as much damage as it does in multiplayer.  This is by design, and makes the guns in multiplayer balanced while still giving you a powerful and dominant one in single player. 

However, this is strange in a co-op game: fusion effectively does half as much damage to robots in co-op as in single player, significantly changing some strategies. 

It would be great if co-op fusion used the single player damage value.  Or, if that upsets applecarts, to at least have an option to play that way.

(Credit: Bagheera)

**Comment by roncli** [comment]
Having just looked at this code recently, this would be an easy fix.  The question is if we want to make it an option or not (I'm leaning towards no.)

**Comment by CDarrow** [comment]
Yes, of course.  This issue is 99% political and 1% technical.  Whereas most have more of a 90/10 split.  We should deal with it in a raft of coop issues, when we have the attention of pilots that care about coop.  That's not this build.
